### ISSUE TITLE
Py2app doesn't work with Pyrebase

### ISSUE THREAD

**Issue by itaymish** [ronaldoussoren/py2app#328]
I tried make a Tkinter app with Pyrebase. It worked well.
I tried to convert it to Mac app, I mentioned that I'm using Pyrebase in the setup.py file, but it didn't worked. It showed me error. What can I do?
I tried to run it normally and it worked.
Can you help me?

**Comment by ronaldoussoren** [comment]
Does adding pyrebase to the packages option help ("python3 setup.py py2app --packages=pyrebase", or

```
setup(
    ...
    options={
      "py2app": {
       "packages": ["pyrebase"]
   )
)
```

**Comment by itaymish** [comment]
I have already tried that.

**Comment by ronaldoussoren** [comment]
This looks to be #1: pyrebase depends on gcloud, and that library uses pkg_resources to find itself (in order to look up its own version number).  Issue #1 is about including relevant metadata in the app bundle.

A workaround is to patch gcloud to not do that (hardcoding a version number), that requires changes to ``__init__.py`` and ``connection.py`` in that package.  When I do that I no longer get a startup error, although I cannot test further because I don't have a firebase account.

The easiest way to debug startup problems is to start the app in the Terminal, given "hello.app" you can start using "dist/hello.app/Contents/MacOS/hello".  This will print any errors to the Terminal, instead of those errors being swallowed by the system.

**Comment by ronaldoussoren** [comment]
This problem should be fixed in the upcoming 0.26 (you can test by installing from this repository).   

I expect to release later this week, after spending some more time on working my way through the issue backlog.
